nginx_reloader.py

```diff
@@ -63,19 +75,23 @@ def populate_hosts(fname):
         if not validate_ip(s):
           servers.append(s)
   return servers
-        
+
 def resolve_dns(servers):
   servers_dict = {}
   myResolver = dns.resolver.Resolver() #create a new instance named 'myResolver'
   for server in servers:
+    if len(include_list) > 0 and server not in include_list:
```

Comment:
* Let's say you have 50 servers and need to only monitor 10, use the include option.
* Let's say you have 50 servers and need to monitor 40, use the exclude option.
* If you have to monitor all the upstreams then just use without include or exclude option